docs/spec/argv.md

```
@@ -18,6 +18,38 @@ measurement, checked on every run rather than asserted here — see
 :::
 
+## Seeing it for yourself
+
+`usage explain` answers this page for one command line: which token bound to
+which flag or argument, and where every value that no token supplied came from.
+
+```sh
+usage explain -f mycli.usage.kdl -- mycli -j8 --env=prod build a -- --raw
+```
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)